ViteMaDose/Helpers/Utils/LocalizedString.swift

```diff
@@ -0,0 +1,131 @@
+        }
+
+    }
+
+    enum CountrySelection {
+
+        enum Title {
+
+            static let title = "country_selection.title.title".localized()
+            static let highlightedText = "country_selection.title.highlighted_text".localized()
+
+        }
+
+    }
+
+    enum VaccinationLocationsList {
+
+        static let listTitle = "vaccination_locations_list.list_title".localized()
+        static let foundLocations = "vaccination_locations_list.found_locations".localized()
+        static let foundLocation = "vaccination_locations_list.found_location".localized()
+        static let noAppointments = "vaccination_locations_list.no_appointments".localized()
+        static let dateUnavailable = "vaccination_locations_list.date_unavailable".localized()
+        static let openRoute = "vaccination_locations_list.open_route".localized()
+        static let bookButton = "vaccination_locations_list.book_button".localized()
+        static let verifyButton = "vaccination_locations_list.verify_button".localized()
+        static let locationNameUnavailable = "vaccination_locations_list.location_name_unavailable".localized()
+        static let addressUnavailable = "vaccination_locations_list.address_unavailable".localized()
+        static let cancelButton = "vaccination_locations_list.cancel_button".localized()
+
+        static func date(date: String, time: String) -> String {
+            return "vaccination_locations_list.date".localized().format([date, time])
+        }
+        static func dosesCount(_ quantity: Int) -> String {
+            if quantity > 1 {
+                return PluralHandler.doses(quantity)
+            } else {
+                return PluralHandler.dose(quantity)
+            }
+        }
+        static func availableLocations(_ plural: String) -> String {
+            return "vaccination_locations_list.available_locations".localized().format([plural])
+        }
+        static func allLocations(_ plural: String) -> String {
+            return "vaccination_locations_list.all_locations".localized().format([plural])
+        }
+        static func lastUpdate(date: String, time: String) -> String {
+            return "vaccination_locations_list.last_update".localized().format([date, time])
+        }
+        static func title(quantity: String, area: String) -> String {
+            return "vaccination_locations_list.title".localized().format([quantity, area])
+        }
+        static func noResults(area: String) -> String {
+            return "vaccination_locations_list.no_results".localized().format([area])
+        }
+
+    }
+
+    enum GenericError {
+
+        static let cancelButton = "generic_error.cancel_button".localized()
+        static let title = "generic_error.title".localized()
+        static let retryButton = "generic_error.retry_button".localized()
+        static let defaultMessage = "generic_error.default_message".localized()
+
+    }
+
+    fileprivate enum PluralHandler {
```

Comment:
@victor-sarda Just need you to see if this is okay for you. If it is, it's mergeable.

Reply:
In any case, this PR will not be merged into develop but in a dedicated branch, making a lot of modifications possible before the final PR.

Reply:
If he wants he can push a commit to your branch directly before merging it.
Also he told me he was making a branch with your changes, adding some cleanup